packages/sfui/frameworks/react/components/SfInput/SfInput.tsx

```diff
@@ -4,9 +4,9 @@ import { SfInputSize, useFocusVisible, type SfInputProps, polymorphicForwardRef
 const defaultWrapperTag = 'span';
 
 const sizeClasses = {
-  [SfInputSize.sm]: 'py-1.5 px-4',
-  [SfInputSize.base]: 'py-2 px-4',
-  [SfInputSize.lg]: 'py-3 px-4',
+  [SfInputSize.sm]: 'py-1.5 px-4 max-h-[32px]',
+  [SfInputSize.base]: 'py-2 px-4 max-h-[40px]',
+  [SfInputSize.lg]: 'py-3 px-4 max-h-[48px]',
```

Comment:
I think combining padding and max height does not completely resolve the issue. Even if you limit the height padding is still there and limits the content. As suggested in the ticket, I think it'd be better to set a fixed height (e.g. `h-[32px]`) and remove vertical padding (`py`).
Notice that `px-4` class doesn't depend on the input size, so you can move it to general classNames.

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2714/commits/8dd325e2088a78e8adf30fa26355c741fb7ff88e